README.md

```diff
@@ -1,369 +1,177 @@
-```
-Exit `nano` by Ctrl-O and Ctrl-X  
+## Getting Started
+
+The following is an example setup using the POA Sokol testnet as the Home network, and the Ethereum Kovan testnet as the Foreign network. The instructions for the Bridge UI are identical for an `ERC20-to-ERC20` configuration, but the smart contract deployment steps will vary.
 
-* mkdir `keys`
-* put validator JSON keystore into `keys` folder
-* create `password.txt` file and insert your password for the key
+### Dependencies
+
+- [poa-bridge-contracts](https://github.com/poanetwork/poa-bridge-contracts)
+- [token-bridge](https://github.com/poanetwork/token-bridge)
+- [node.js](https://nodejs.org/en/download/)
+- [Nifty Wallet](https://chrome.google.com/webstore/detail/nifty-wallet/jbdaocneiiinmjbjlgalhcelgbejmnid?hl=en) or [MetaMask](https://metamask.io/)
+
+### Example Setup
+
+1. Create an empty folder for setting up your bridge. In this example we call it `sokol-kovan-bridge`.
+`mkdir sokol-kovan-bridge && cd sokol-kovan-bridge`  
 
-10. At this step your folder structure should look like this
+2. Prepare temporary ETH address(es) for deployment by creating new account(s) in Nifty Wallet or MetaMask. See the [wallet resources](#resources) if you need more information on this step. This account is used:
+    * for deploying bridge contracts to both networks
+    * as the bridge contracts management wallet
+    * as the validator's wallet address(es)
+
+3. Fund the test account(s).
+    * Fund Home account(s) using the [POA Sokol Faucet](https://faucet-sokol.herokuapp.com/)
+    * Get free Kovan Coins from the [gitter channel](https://gitter.im/kovan-testnet/faucet) or [Iracus faucet](https://github.com/kovan-testnet/faucet) for Foreign account(s). Get 5 Keth to 1 acc, and transfer from there to all other wallets if more than one account is used.
+
+4. Deploy the Sokol <-> Kovan Bridge contracts.
+    * Go to the the `sokol-kovan-bridge` folder created in step 1 and `git clone https://github.com/poanetwork/poa-bridge-contracts`
+    * Follow instructions in the [POA Bridge contracts repo](https://github.com/poanetwork/poa-bridge-contracts).
+    * Set the parameters in the .env file.
+      * `DEPLOYMENT_ACCOUNT_PRIVATE_KEY`: Export the private key from step 2
+      * `HOME_RPC_URL`=https://sokol.poa.network
+      * Wallet address(es) for bridge contracts management. For testing, you can use the same address for all address values in the file. This includes:
+        * `HOME_OWNER_MULTISIG`
+        * `HOME_UPGRADEABLE_ADMIN_VALIDATORS`
+        * `HOME_UPGRADEABLE_ADMIN_BRIDGE`
+        * `FOREIGN_OWNER_MULTISIG`
+        * `FOREIGN_UPGRADEABLE_ADMIN_VALIDATORS`
+        * `FOREIGN_UPGRADEABLE_ADMIN_BRIDGE`
+        * `VALIDATORS` _Note: Wallet address(es) for validator(s) are separated by a space. For testing, you can use the same address that was used as the bridge contracts management account._
+      * `FOREIGN_RPC_URL`=https://kovan.infura.io/mew
```

Comment:
not sure if https://kovan.infura.io/mew needs the mew. This was from the previous version. Feel free to eliminate, I believe there are 2 instances in these examples.

Reply:
it is not an issue - this url is used by MyEthereumWallet application. So, we can leave this as is.